packages/x-data-grid-premium/src/hooks/features/aiAssistant/utils.ts

```diff
@@ -0,0 +1,7 @@
+import { DataGridPremiumProcessedProps } from '../../../models/dataGridPremiumProps';
+
+export const isAiAssistantAvailable = (
```

Comment:
Nitpick: Is it needed? 🙈 

I feel the latter would be cleaner.

```diff
-if (isAiAssistantAvailable(props)) {
+if (props.aiAssistant) {
```

Or do we plan to add more conditions to it later?